--- a/src/Geocoder.php
+++ b/src/Geocoder.php
@@ -103,9 +103,14 @@
   /**
    * {@inheritdoc}
    */
-  public function reverse(string $latitude, string $longitude, array $providers): ?AddressCollection {
+  public function reverse($latitude, $longitude, array $providers): ?AddressCollection {
     // Allow other modules to adjust the coordinates.
     $this->moduleHandler->alter('reverse_geocode_coordinates', $latitude, $longitude);
+
+    if ($latitude === NULL || $longitude === NULL) {
+      return NULL;
+    }
+
     /** @var \Drupal\geocoder\GeocoderProviderInterface $provider */
     foreach ($providers as $provider) {
       try {
--- a/src/GeocoderInterface.php
+++ b/src/GeocoderInterface.php
@@ -35,6 +35,6 @@
    *
    * @return \Geocoder\Model\AddressCollection|null
    */
-  public function reverse(string $latitude, string $longitude, array $providers): ?AddressCollection;
+  public function reverse($latitude, $longitude, array $providers): ?AddressCollection;
 
 }
--- a/src/ProviderUsingHandlerBase.php
+++ b/src/ProviderUsingHandlerBase.php
@@ -131,6 +131,9 @@
    * @throws \Geocoder\Exception\Exception
    */
   protected function doReverse($latitude, $longitude) {
+    if ($latitude === NULL || $longitude === NULL) {
+      return NULL;
+    }
     return $this->getHandlerWrapper()->reverse($latitude, $longitude);
   }
 
--- a/src/ConfigurableProviderUsingHandlerWithAdapterBase.php
+++ b/src/ConfigurableProviderUsingHandlerWithAdapterBase.php
@@ -105,6 +105,9 @@
    * {@inheritdoc}
    */
   protected function doReverse($latitude, $longitude) {
+    if ($latitude === NULL || $longitude === NULL) {
+      return NULL;
+    }
     $this->throttle->waitForAvailability($this->pluginId, $this->configuration['throttle'] ?? []);
     return parent::doReverse($latitude, $longitude);
   }
