--- a/mailjet.module
+++ b/mailjet.module
@@ -472,6 +472,9 @@
     ];
 
     $mailjetApiClient = mailjet_new();
+    if (!$mailjetApiClient) {
+        return;
+    }
     $mj_lists = [];
     $mj_lists[] = mailjet_get_default_list_id($mailjetApiClient);
     $properties = [];
@@ -547,7 +550,9 @@
  */
 function mailjet_get_default_list_id($mailjetApiClient)
 {
-    $mailjetApiClient = mailjet_new();
+    if (!$mailjetApiClient) {
+        return 0;
+    }
     // Gets Mailjet contact list by name
     $contactsList = \Drupal\mailjet\MailjetApi::getMailjetContactListByName(mailjet_get_default_list_name());
 
@@ -568,6 +573,9 @@
 function mailjet_create_default_list()
 {
     $mailjetApiClient = mailjet_new();
+    if (!$mailjetApiClient) {
+        return 0;
+    }
     $responseNewContactsList = \Drupal\mailjet\MailjetApi::createMailjetContactList(mailjet_get_default_list_name());
     if (false != $responseNewContactsList) {
         $def_list_id = $responseNewContactsList[0]['ID'];
@@ -625,6 +633,9 @@
 {
     $contact_lists = [];
     $mailjetApiClient = mailjet_new();
+    if (!$mailjetApiClient) {
+        return $contact_lists;
+    }
     $contactsListResponse = \Drupal\mailjet\MailjetApi::getMailjetContactLists(get_mailjet_get_lists_limit());
     if (false != $contactsListResponse) {
         foreach ($contactsListResponse as $list) {
@@ -640,6 +651,9 @@
     //get existing contact properties from MailJet
     $properties = [];
     $mailjetApiClient = mailjet_new();
+    if (!$mailjetApiClient) {
+        return $properties;
+    }
     $response = \Drupal\mailjet\MailjetApi::getMailjetContactProperties();
     if (false != $response) {
         foreach ($response as $property) {
