diff --git a/mailjet.module b/mailjet.module
index d2b8182..1169d0d 100644
--- a/mailjet.module
+++ b/mailjet.module
@@ -461,6 +461,11 @@ function mailjet_user_delete(User $user)
  */
 function mailjet_sync_single_user(User $user, $action)
 {
+    global $config;
+    if (isset($config['disable_mailjet_sync']) && $config['disable_mailjet_sync']) {
+        return;
+    }
+
     $config_mailjet = Drupal::config('mailjet.settings');
 
     if (empty($user->get('mail')->value)) {
